src/archive.rs

```diff
@@ -241,10 +243,11 @@ impl<'a> EntriesFields<'a> {
         }
 
         let file_pos = self.next;
-        let size = header.entry_size()?;
-
+        let mut size = header.entry_size()?;
+        if (size == 0) && (pax_size.is_some()) {
+            size = pax_size.unwrap();
+        }
```

Comment:
I think this is dead code now?

Reply:
`size` is needed to advance the `self.next` pointer

Reply:
Also this governs the `data` size in `EntryFields`